LICENSE

```diff
@@ -1,116 +1,5 @@
-CC0 1.0 Universal
+Copyright (C) 2018-2019 by Jonas Schievink
```

Comment:
"by" is probably not a good idea in a copyright notice. See https://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/ for how this should look in open-source projects.

Reply:
Interesting read, done